modules/cluster-logging-deploy-eo-cli.adoc

```diff
@@ -0,0 +1,198 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-deploying.adoc
+
+[id="cluster-logging-deploy-eo-cli_{context}"]
+= Install the Elasticsearch Operator using the CLI
+
+You *must* install the Elasticsearch Operator using the CLI following the directions below.
+
+.Prerequisites
+
+Ensure that you have the necessary persistent storage for Elasticsearch. Note that each Elasticsearch node
+requires its own storage volume.
+
+Elasticsearch is a memory-intensive application. Each Elasticsearch node needs 16G of memory for both memory requests and limits.
+The initial set of {product-title} nodes might not be large enough to support the Elasticsearch cluster. You must add additional nodes to the
+{product-title} cluster to run with the recommended or higher memory. Each Elasticsearch node can operate with a lower
+memory setting though this is not recommended for production deployments.
+
+.Procedure
+
+. Create a Namespace for the Elasticsearch Operator.
+
+.. Create a Namespace object YAML file (for example, `eo-namespace.yaml`) for the Elasticsearch Operator:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-operators-redhat <1>
+  annotations:
+    openshift.io/node-selector: ""
+  labels:
+    openshift.io/cluster-monitoring: "true" <2>
+----
+<1> You must specify the `openshift-operators-redhat` namespace. To prevent 
+possible conflicts with metrics, you should configure the Prometheus Cluster 
+Monitoring stack to scrape metrics from the `openshift-operators-redhat` 
+namespace and not the `openshift-operators` namespace. The `openshift-operators` 
+namespace can contain community operators, which are untrusted and could publish 
```

Comment:
Unclear if "can" contain means it "might" contain or "is allowed to" contain. Also, "operators" is lowercase because they are community and not OCP?

Reply:
outdated